command/s3_put.go

```diff
@@ -229,10 +239,10 @@ retryLoop:
 		case <-ctx.Done():
 			return errors.New("s3 put operation canceled")
 		case <-timer.C:
-			filesList := []string{s3pc.LocalFile}
+			filesList := []string{filepath.Join(s3pc.workDir + s3pc.LocalFile)}
```

Comment:
the directory and file name should be separate parameters

Reply:
sorry, fixed